SpatialGDK/Documentation/content/get-started/example-project/exampleproject-cloud-deployment.md

```diff
@@ -15,10 +15,12 @@ You prepare both your server-workers and client-workers for upload to the cloud
 
 <%(/Expandable)%>
 
-**Schema and snapshot**</br>
+**Schema and snapshot**
+
 As you already generated schema and a snapshot for the Example Project when you launched a local deployment, and haven’t changed the project since, you don’t need to generate a schema or snapshot. However, when you do make any changes to the project, you must generate schema and create a snapshot before launching a local or cloud deployment. 
```

Comment:
"As you have already generated" ?

Sorry! I know this isn't your text. We should have an acronym for that (IKTIYT) but I think it's worth fixing. Very clunky otherwise.